src/utils.test.js

```diff
@@ -1,88 +1,158 @@
     }
 
     const formatted = formatPinoBrowserLogEvent(pinoBrowserLogEvent)
 
     expect(formatted).toEqual({
-        metadata: {
-          url: 'http://localhost/',
-          level: 'info',
-          child1: 'value1',
-          child2: 'value2',
-          child3: 'value3',
-          browser: true
-        },
-        log_entry: 'a message from pino logger',
-        timestamp: 1593372837388
-      }
-    )
+      metadata: {
+        url: "http://localhost/",
+        level: "info",
+        child1: "value1",
+        child2: "value2",
+        child3: "value3",
+        browser: true,
+      },
+      log_entry: "a message from pino logger",
+      timestamp: 1593372837388,
+    })
     done()
   })
 
   it("correctly logs metadata for string and object messages", async (done) => {
     const pinoBrowserLogEvent = {
       ts: 1593372837388,
-      messages: [{c: 3}, "a message", "from pino logger", {b: 2}], bindings: [{a: 1}], level: {value: 30}
+      messages: [{ c: 3 }, "a message", "from pino logger", { b: 2 }],
+      bindings: [{ a: 1 }],
+      level: { value: 30 },
     }
     const formatted = formatPinoBrowserLogEvent(pinoBrowserLogEvent)
 
     expect(formatted).toEqual({
-        metadata: {
-          url: 'http://localhost/',
-          level: 'info',
-          a: 1,
-          b: 2,
-          c: 3,
-          browser: true
-        },
-        log_entry: 'a message from pino logger',
-        timestamp: 1593372837388
-      }
-    )
+      metadata: {
+        url: "http://localhost/",
+        level: "info",
+        a: 1,
+        b: 2,
+        c: 3,
+        browser: true,
+      },
+      log_entry: "a message from pino logger",
+      timestamp: 1593372837388,
+    })
     done()
   })
 
   it("correctly adds logflare transform directives", async (done) => {
     const options = {
       transforms: {
-        numbersToFloats: true
-      }
+        numbersToFloats: true,
+      },
     }
     const pinoBrowserLogEvent = {
       ts: 1593372837388,
       messages: ["a message", "from pino", "logger"],
-      bindings: [{child1: "value1"}, {child2: "value2"}, {child3: "value3"}],
-      level: {value: 30, label: "info"}
+      bindings: [
+        { child1: "value1" },
+        { child2: "value2" },
+        { child3: "value3" },
+      ],
+      level: { value: 30, label: "info" },
     }
 
     const formatted = formatPinoBrowserLogEvent(pinoBrowserLogEvent)
     const withTransforms = addLogflareTransformDirectives(formatted, options)
 
     expect(withTransforms).toEqual({
+      metadata: {
+        url: "http://localhost/",
+        level: "info",
+        child1: "value1",
+        child2: "value2",
+        child3: "value3",
+        browser: true,
+      },
+      log_entry: "a message from pino logger",
+      timestamp: 1593372837388,
+      "@logflareTransformDirectives": {
+        numbersToFloats: true,
+      },
+    })
+    done()
+  })
+
+  describe("toLogEntry", () => {
```

Comment:
Prettier formatted this file when I saved it, so everything above this line (excluding the `import` of course) is all just formatting changes. I'm assuming Prettier was added to this project after this file was created, and these files didn't end up getting formatted when that happened. Hopefully you're okay with this!

Everything below this line, include this line, was added.